nullaway/src/main/java/com/uber/nullaway/fixserialization/out/SuggestedFixInfo.java

```diff
@@ -87,7 +87,7 @@ public String tabSeparatedToString() {
         + '\t'
         + (enclosingInfo.getClazz() == null
             ? "null"
-            : ASTHelpers.getSymbol(enclosingInfo.getClazz()))
+            : ASTHelpers.getSymbol(enclosingInfo.getClazz()).flatName())
```

Comment:
Hi @lazaroclapp, this only changes the class name for the enclosing class of the symbol triggering the fix. The actual class name for the fix should be changed as well. They are located in `tabSeparatedToString()` methods of subclasses of `AbstractFixLocation` and each `enclosingClass` should be changed to `enclosingClass.flatName()`.

Reply:
I will take a quick look at this

Reply:
@nimakarimipour fixed in [1fe3cc7](https://github.com/uber/NullAway/pull/592/commits/1fe3cc7e3aa967f32f2f334edfb8eb1dce98b09c).  But none of the tests failed; so we need better test coverage of this scenario

Reply:
Can you think of some test cases here that would apply, @nimakarimipour, and add them to this PR?

Reply:
I added some extra checks in [31dd46b](https://github.com/uber/NullAway/pull/592/commits/31dd46b2e9ce1bc47ea50a76d5bda8cefafbb110) for one of the tests.  I'm moderately confident this is sufficient, and that there is no need to add these fix display tests everywhere (since this checks the same flat name is always getting printed).  @lazaroclapp what do you think?  If you want me to change the other tests also let me know.

(I'm also not happy with the amount of copy-paste in the latest commit but we can refactor this test code in a follow-up.)